src/main/kotlin/com/cognifide/gradle/aem/common/AemExtension.kt

```diff
@@ -341,6 +342,13 @@ open class AemExtension(@Internal val project: Project) {
 
     fun temporaryDir(name: String) = AemTask.temporaryDir(project, name)
 
+    @Nested
+    val tailConfig = TailConfig()
```

Comment:
why is it inside `AemExtension` , not `Tail` task ?

Reply:
fixed